lib/runtime/GetChunkFilenameRuntimeModule.js

```diff
@@ -119,6 +119,9 @@ class GetChunkFilenameRuntimeModule extends RuntimeModule {
 			 */
 			const unquotedStringify = value => {
 				const str = `${value}`;
+				if (typeof value === "number") {
```

Comment:
Adding the dynamic `import()` failed here, it was trying to do `unquotedStringify(2)` which would return empty string, that would failed down in staticUrls.

Reply:
Line 121 should convert the number into a string. I don't think this extra check is needed.

Reply:
Yep, you're right. The real problem is the `-2` on the slice.


lets consider
```es6
c.name = undefined
c.id = 2
...
  name: unquotedStringify(c.name || c.id), // line 148
...
```

Result:
```
> unquotedStringify(2)
''
```


```es6
const unquotedStringify = value => {
  const str = `${value}`; // str => '2'
  if (str.length >= 5 && str === `${c.id}`) { // str.length === 1, so this is false
    // This is shorter and generates the same result
    return '" + chunkId + "';
  }
  const s = JSON.stringify(str); // s => '"2"', s.length = 3
  return s.slice(1, s.length - 2); // s.slice(1, 1) => '' - it takes from char at index 1, to char index 1 (not inclusive, so index 0)
};
```